### PR TITLE
Fixed package to run with ros2 run

### PR DESCRIPTION
**Pull request by ahcorde**
Fixed package to run with `ros2 run rqt_action rqt_action`. Related to this issue https://github.com/ros-visualization/rqt_console/issues/20

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Comment by cottsay** [comment]
Can you elaborate on how this solves the problem? Does it end up installing the script twice?

**Comment by ahcorde** [comment]
I added an entrypoint and removed the script instalation otherwise this is not working on Windows

File: setup.py [hunk 1]
```diff
@@ -52,4 +51,9 @@
     ),
     license='BSD',
     tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+            'rqt_reconfigure = ' + package_name + '.main:main',
```
Comment (cottsay) [inline setup.py:56]:
```suggestion
            package_name + ' = ' + package_name + '.main:main',
```

Comment (cottsay) [reply setup.py:56]: *PING* - let's get this wrapped up.

Comment (ahcorde) [reply setup.py]: Fixed https://github.com/ros-visualization/rqt_reconfigure/pull/81/commits/1907d8d530752cac71ed73103cff5123b37f06bc

**Comment by ahcorde** [comment]
Running CI up to `rqt_reconfigure`

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10920/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6270/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8892/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10822/)

**Comment by dirk-thomas** [comment]
With this change you can't invoke `rqt_reconfigure` anymore (without `ros2 run`) so this breaks existing use cases and documentation.

**Comment by cottsay** [comment]
> With this change you can't invoke `rqt_reconfigure` anymore (without `ros2 run`) so this breaks existing use cases and documentation.

In what ROS version was this possible? I don't see how `lib/rqt_reconfigure` was part of your `PATH` before this change.

**Comment by dirk-thomas** [comment]
> In what ROS version was this possible? I don't see how `lib/rqt_reconfigure` was part of your `PATH` before this change.

You are right. This package only offered the executable in `lib/rqt_reconfigure` - so not a regression. Other packages offer the executable in both places for user convenience so I falsely assumed this package would do the same.

**Pull request merged**